React/Views/RCTPicker.h

```diff
@@ -18,7 +18,9 @@
 @property (nonatomic, assign) NSInteger selectedIndex;
 
 @property (nonatomic, strong) RCTUIColor *color; // TODO(OSS Candidate ISS#2710739)
+#if !TARGET_OS_OSX // [TODO(OSS Candidate ISS#2710739) // NSControl defines font prop for macOS, but iOS superviews don't
```

Comment:
For my own sake: Where can I see the list of ISS bugs / where they are filed?

Reply:
There's an Issues tab at the top of the repo that shows them all https://github.com/microsoft/react-native-macos/issues

Reply:
This is a pretty generic bug we use to track diffs in our fork and actually come to think of it, these should all probably get fixed up with GitHub bugs instead of ADO ones

Reply:
Yeah I think we should stop attributing to the ADO one for sure - attributing to one single item doesn't have too much value and doesn't help during merges at all since if you're using a proper 3-way merge you don't need the additional context, your merge tool already will tell you what you've changed locally vs. what got changed in the other branch. I would vote towards not using these tags unless we can tag it with a github issue that is specific and maybe a good "first issue" candidate or we add comments to explain the divergence